.eslintrc.json

```diff
@@ -36,6 +36,8 @@
         "semi": [
             "error",
             "always"
-        ]
+        ],
+        "no-mixed-spaces-and-tabs": [0] // disable rule
```

Comment:
Shouldn't be disabling rules. Just run `npx eslint . --fix`. Should fix linting errors

Reply:
I have removed the rule